test/Microsoft.ML.Functional.Tests/Explainability.cs

```diff
@@ -49,6 +51,43 @@ public void GlobalFeatureImportanceWithPermutationFeatureImportance()
                 Common.AssertMetricsStatistics(metricsStatistics);
         }
 
+        /// <summary>
+        /// GlobalFeatureImportance: PFI can be used to compute global feature importance. Here it is used with a model loaded from disk.
+        /// </summary>
+        [Fact]
+        public void GlobalFeatureImportanceWithPermutationFeatureImportanceWithLoadedModel()
+        {
+            var mlContext = new MLContext(seed: 1);
+
+            // Get the dataset
+            var data = mlContext.Data.LoadFromTextFile<HousingRegression>(GetDataPath(TestDatasets.housing.trainFilename), hasHeader: true);
+
+            // Create a pipeline to train on the housing data.
+            var pipeline = mlContext.Transforms.Concatenate("Features", HousingRegression.Features)
+                .Append(mlContext.Regression.Trainers.Sdca());
+
+            // Fit the pipeline and transform the data.
+            var model = pipeline.Fit(data);
+
+            var modelAndSchemaPath = GetOutputPath("TestFunctionalTestPFI.zip");
+            mlContext.Model.Save(model, data.Schema, modelAndSchemaPath);
+
+            ITransformer loadedModel;
+            using (var fs = File.OpenRead(modelAndSchemaPath))
```

Comment:
You aren't using `fs` here. This can be removed.

Reply:
Fixed in my last commit.